src/connector/src/common.rs

```diff
@@ -386,6 +386,47 @@ impl NatsCommon {
         Ok(subscription)
     }
 
+    pub(crate) async fn build_consumer(
+        &self,
+        start_sequence: Option<u64>,
+    ) -> anyhow::Result<
+        async_nats::jetstream::consumer::Consumer<async_nats::jetstream::consumer::pull::Config>,
+    > {
+        let context = self.build_context().await?;
+        let stream = self.build_or_get_stream(context.clone()).await?;
+        let name = "risingwave-consumer";
+        match start_sequence {
+            Some(v) => {
+                let consumer = stream
+                    .get_or_create_consumer(
+                        name,
+                        async_nats::jetstream::consumer::pull::Config {
+                            durable_name: Some(name.to_string()),
```

Comment:
since we have no mechanism to ack messages, I am wondering whether we really need to make the client durable. In recovery or split change, we always open a new client and there is no need to let the server keep the consumption progress. 

> Setting durable_name to Some(...) will cause this consumer to be “durable”. This may be a good choice for workloads that benefit from the JetStream server or cluster remembering the progress of consumers for fault tolerance purposes. If a consumer crashes, the JetStream server or cluster will remember which messages the consumer acknowledged. When the consumer recovers, this information will allow the consumer to resume processing where it left off. If you’re unsure, set this to Some(...).
> Setting durable_name to None will cause this consumer to be “ephemeral”. This may be a good choice for workloads where you don’t need the JetStream server to remember the consumer’s progress in the case of a crash, such as certain “high churn” workloads or workloads where a crashed instance is not required to recover [source](https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html#structfield.durable_name)

Reply:
+1

Reply:
Removed it.